api/public/v2/commit/serializers.py

```diff
@@ -47,3 +53,39 @@ class CommitDetailSerializer(CommitSerializer):
     class Meta:
         model = Commit
         fields = CommitSerializer.Meta.fields + ("report",)
+
+
```

Comment:
A subset of fields of the `ReportSession` model, matching the contents of old v1 sessions endpoint, except keys are descriptive instead of single letters from before.